permissions/plugin-ivytrigger.yml

```diff
@@ -6,6 +6,4 @@ issues:
 paths:
   - "org/jenkins-ci/plugins/ivytrigger"
 developers:
-  - "gbois"
-  - "bbergquist"
```

Comment:
Reverts https://github.com/jenkins-infra/repository-permissions-updater/pull/504.

Last Jira activity was in 2017, and there seems to have never been activity in the plugin.